src/components/Calculator.js

```diff
@@ -31,8 +32,25 @@ class Calculator extends Component {
 
   render() {
     const { data } = this.state;
-    return data.map((value, i) => <div className={`item${i}`} key={`item${i + 1}`}>{value}</div>);
+    const { total, handler } = this.props;
+    const mytype = 'button';
+    return data.map((value, i) => {
+      if (i === 0) {
+        return <input type={mytype} readOnly className={`item${i}`} key={`item${i + 1}`} value={total} />;
+      }
+
+      return <input type={mytype} onClick={handler} className={`item${i}`} key={`item${i + 1}`} value={value} />;
+    });
```

Comment:
- [ ] Kindly when I try to click on a number it doesn't show up on the screen, it should click on an operation button after that it gets displayed and it shows 2 errors when I click on any numbers, Kindly update that behavior, so when I click on a number it gets displayed directly and without any errors.